packages/plugin-dom-renderer/DomRendererPlugin.ts

```diff
@@ -2,5 +2,5 @@ import { JWPlugin } from '../core/src/JWPlugin';
 import { DomRenderer } from './DomRenderer';
 
 export class DomRendererPlugin extends JWPlugin {
-    static readonly renderers = [DomRenderer];
+    static readonly renderers = [new DomRenderer()];
```

Comment:
@Zinston Just so you know, if you make Renderer an interface (as suggested by nby) rather than a class, it would be possible to write:
```export class DomRendererPlugin extends JWPlugin implements Renderer {```

Such that a plugin can be a renderer, rather than serving no other purpose than providing a `renderers` key to another object.

You can even implements multiple interfaces, for example:
```export class MyPlugin extends JWPlugin implements Renderer, WhateverFeature, WhateverOtherFeature {```

Reply:
This particular plugin only does that but nothing should stop you from making a plugin that declares renderers _and_ does other things too... Also, I made `Renderer` a class so that it could handle `addRenderingFunction` and we wouldn't have to rewrite it for each renderer.